www/package.json

```diff
@@ -21,6 +22,8 @@
   "engines": {
     "node": ">=0.8.0"
   },
-  "scripts": {},
+  "scripts": {
+    "install": "./node_modules/grunt-cli/bin/grunt browserify"
```

Comment:
This line makes the build fail, at least with a local fh-digger. Using `grunt browserify` alone seems to work fine